Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build install test clean docker-build docker-push help
+.PHONY: build install test clean docker-build docker-push lint lint-fix help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)